indra/tests/test_omnipath.py

```diff
@@ -0,0 +1,60 @@
+import requests
+from indra.sources.omnipath import OmniPathProcessor
+from indra.sources.omnipath.api import op_url
+from indra.statements import Agent
+from indra.preassembler.grounding_mapper import GroundingMapper
```

Comment:
Same as in the processor, we should now use `from indra.ontology.standardize` import standardize_agent_name. Given that the ontology is mocked on Travis, we may need to add some more entries to the mock ontology to make tests pass.